NullPointerException: Attempt to write to field 'String FeedsCredentials.userID' on a null object reference

from gplay dev console:

```
java.lang.RuntimeException: An error occured while executing doInBackground()
	at android.os.AsyncTask$3.done(AsyncTask.java:304)
	at java.util.concurrent.FutureTask.finishCompletion(FutureTask.java:355)
	at java.util.concurrent.FutureTask.setException(FutureTask.java:222)
	at java.util.concurrent.FutureTask.run(FutureTask.java:242)
	at android.os.AsyncTask$SerialExecutor$1.run(AsyncTask.java:231)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1112)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:587)
	at java.lang.Thread.run(Thread.java:818)
Caused by: java.lang.NullPointerException: Attempt to write to field 'java.lang.String fr.gaulupeau.apps.Poche.data.FeedsCredentials.userID' on a null object reference
	at fr.gaulupeau.apps.Poche.network.WallabagServiceEndpointV2.getCredentials(WallabagServiceEndpointV2.java:92)
	at fr.gaulupeau.apps.Poche.network.WallabagService.getCredentials(WallabagService.java:57)
	at fr.gaulupeau.apps.Poche.network.tasks.GetCredentialsTask.doInBackground(GetCredentialsTask.java:32)
	at fr.gaulupeau.apps.Poche.network.tasks.GetCredentialsTask.doInBackground(GetCredentialsTask.java:10)
	at android.os.AsyncTask$2.call(AsyncTask.java:292)
	at java.util.concurrent.FutureTask.run(FutureTask.java:237)
	... 4 more
```



App-Version     26

Android-Version    Android 5.11

Gerät    Nexus 7 (2012) (grouper)

--- a/app/src/main/java/fr/gaulupeau/apps/Poche/network/WallabagServiceEndpointV2.java
+++ b/app/src/main/java/fr/gaulupeau/apps/Poche/network/WallabagServiceEndpointV2.java
@@ -89,7 +89,7 @@ public class WallabagServiceEndpointV2 extends WallabagServiceEndpoint {
         FeedsCredentials fc = getCredentials("/config", CREDENTIALS_PATTERN);
         // overwrite userID with username because first matcher group of previous regex, which
         // should return the user name, might include the subdirectory in which wallabag is installed
-        fc.userID = username;
+        if(fc != null) fc.userID = username;
         return fc;
     }
 

